File: Find-Resultant-Array-After-Removing-Anagrams/Find-Resultant-Array-After-Removing-Anagrams.py
words = ["abba","baba","bbaa","cd","cd"]
# so sánh từng phần tử hiện tại và phần tử phía sau
def removeAnagrams(words):
    lists = [words[0]]
    for i in range(1,len(words),1):
        if sorted(words[i]) == sorted(words[i-1]):
            ''
        else:
            lists.append(words[i])
    return lists

print(removeAnagrams(words))

File: Find-Resultant-Array-After-Removing-Anagrams/Find-Resultant-Array-After-Removing-Anagrams.js
let words = ["abba","baba","bbaa","cd","cd"]
// thêm vào set và so sánh từng phần tử trong mảng

var removeAnagrams = function(words) {
    let mySet = new Set();
    let list = []
    for(let i = 0 ; i<words.length;i++){
        if(!mySet.has(words[i].split('').sort().join(''))){
            mySet.clear()
            mySet.add(words[i].split('').sort().join(''))
            list.push(words[i])
        }
    }
    return list
};

console.log(removeAnagrams(words))